dd-java-agent/agent-profiling/profiling-controller-openjdk/src/main/java/com/datadog/profiling/controller/openjdk/events/SmapEntryFactory.java

```diff
@@ -61,6 +72,11 @@ public static void registerEvents() {
         annotatedMapsAvailable = false;
       }
     }
+    if (annotatedMapsAvailable) {
+      log.info("Smap entry events registered successfully");
```

Comment:
This should rather be debug. 
```suggestion
      log.debug("Smap entry events registered successfully");
```